packages/core/src/components/header/header.tsx

```diff
@@ -134,13 +140,26 @@ export class Header {
   }
 
   setDropDownMenu() {
+    this.dropdownOptionsOffset = this.calculateDropdownOptionsOffset();
```

Comment:
kan dit niet beter na r.149? (met een `if(this.showDropDown)`)

Reply:
En is het uberhaupt wel nodig hier als het ook altijd bij de didRender berekend wordt?

Reply:
Verplaatst naar de `componentDidLoad`. Alleen bij `componentDidRender` gaat het niet goed als de dropdown wordt geopend na het laden van de pagina.

Reply:
@LukeVerlaan Is deze opmerking nog van toepassing? En zo ja, is dit blokkerend voor het mergen van dit PR?

Reply:
Ok!